config/config-mysql_apache_jvm.yaml

```diff
@@ -0,0 +1,307 @@
+        action: update
+        new_name: agent.googleapis.com/apache/idle_workers
+        operations:
+          - action: aggregate_labels
+            label_set: []
+            aggregation_type: max
+            #this is a hack since the correct value is not necessarily max
+      
+      - metric_name: apache_sent_kilobytes_total
+        action: update
+        new_name: agent.googleapis.com/apache/traffic
+        operations:
+          - action: toggle_scalar_data_type
+
+  metricstransform/host:
+    transforms:
+      # system.cpu.time -> cpu/usage_time
+      - metric_name: system.cpu.time
+        action: update
+        new_name: agent.googleapis.com/cpu/usage_time
+        operations:
+          # change data type from double -> int64
+          - action: toggle_scalar_data_type
+          # change label cpu -> cpu_number
+          - action: update_label
+            label: cpu
+            new_label: cpu_number
+          # change label state -> cpu_state
+          - action: update_label
+            label: state
+            new_label: cpu_state
+          # take mean over cpu_number dimension, retaining only cpu_state
+          - action: aggregate_labels
+            label_set: [ cpu_state ]
+            aggregation_type: mean
+            
+      - metric_name: system.memory.usage
+        action: update
+        new_name: agent.googleapis.com/memory/bytes_used
+        operations:
+          # change data type from int64 -> double
+          - action: toggle_scalar_data_type
+          # aggregate state label values: slab_reclaimable & slab_unreclaimable -> slab (note this is not currently supported)
+          - action: aggregate_label_values
+            label: state
+            aggregated_values: [slab_reclaimable, slab_unreclaimable]
+            new_value: slab
+            aggregation_type: sum
+            
+      - metric_name: system.network.tcp_connections 
+        action: update
+        new_name: agent.googleapis.com/network/tcp_connections
+        operations:
+          # change data type from int64 -> double
+          - action: toggle_scalar_data_type
+          # change label state -> tcp_state
+          - action: update_label
+            label: state
+            new_label: tcp_state
+            
+      - metric_name: system.swap.paging_ops
+        action: update
+        new_name: agent.googleapis.com/swap/io
+        operations:
+          # delete singular type dimension, retaining only direction
+          - action: aggregate_labels
+            label_set: [ direction ]
+            aggregation_type: sum
+
+exporters:
+  stackdriver:  
+    project:  # stackdriver will use the default GCE credentials
+    metric_prefix: # empty to prevent the custom.googleapis.com/ prefix
+
+service:
+  pipelines:
+    # Remember to remove unwanted pipelines from this list
+    metrics/mysql:
+      receivers: [prometheus_exec/mysql]
+      processors: [filter/mysql, resourcedetection, metricstransform/mysql]
```

Comment:
Maybe make resourcedetection the last processor for all pipelines for consistency?